### PR TITLE
feat: Button

### PR DESCRIPTION
**Pull request by manish-GH**
Preview: https://sad-curie-23789d.netlify.app/components/button/

### CODE REVIEW

File: components/Button/styles.css [hunk 1]
```diff
@@ -1 +1,108 @@
...
+.btn-disabled.btn-floating {
+	background-color: var(--theme-primary-lighter);
+}
+
+.btn-disabled:hover {
+	cursor: auto;
+}
+.btn-disabled:hover i {
+	cursor: auto;
+}
```
Comment (sri-rishi) [inline components/Button/styles.css:98]: for disabled button use curser: not-allowed

File: global-styles/reset.css [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 	--theme-warning: #e7b100;
 	--theme-warning-light: #fff2b2;
 	/* --theme-warning: #FBAF00; */
 	--theme-success: #6ab547;
 	--theme-success-light: #c3e1b5;
 	--theme-error: #ee3625;
-	/* --theme-error: #ef4a3a; */
 	--theme-error-light: #f9c2bd;
+	/* --theme-error: #ef4a3a; */
 	/* --theme-error: #eb4943; */
```
Comment (sri-rishi) [inline global-styles/reset.css:16]: remove all comments

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Otherwise code looks goods , just a suggestion  add more types of buttons

**Comment by manish-GH** [comment]
> Otherwise code looks goods , just a suggestion add more types of buttons

I want to be done with the minimum requirements for now. I will add more stuff when I have the time

File: components/Button/styles.css (hunk 1 above)
Comment (manish-GH) [reply components/Button/styles.css:98]: Are you sure about this, I'm verifying because I have never seen this anywhere

**Review by Ankur9669 — APPROVED** [review]
The code looks good to me, you can merge it. Great Work!!!

**Pull request merged**